lib/src/configs/cosim_wrap_config.dart

```diff
@@ -124,9 +126,59 @@ class CosimWrapConfig extends CosimProcessConfig {
           '-f',
           _cosimMakefileName,
         ],
-        environment: environment,
+        environment: {
+          if (environment != null) ...environment!,
+          'PATH':
+              '${Directory(directory).absolute.path}/$_cosimVEnvName/bin:${Platform.environment['PATH']}',
+          'PYTHON3':
+              '${Directory(directory).absolute.path}/$_cosimVEnvName/bin/python3',
+        },
+        // environment: environment,
       );
 
+  /// The name of the requirements.txt that gets generated.
+  static const _cosimRequirementsName = 'requirements.txt';
+
+  /// Generates a requirements.txt file that contains the Python requirements
+  /// for running Cosimulation (namely, cocotb).
+  static void _createRequirementsTxt({
+    required String directory,
+    String cocotbVersion = '1.9.2',
```

Comment:
we should expose this version as a configuration option in the `CosimWrapConfig`?